rain_api_core/egress_util.py

```diff
@@ -47,6 +56,13 @@ def get_presigned_url(session, bucket_name, object_name, region_name, expire_sec
         "X-Amz-SignedHeaders=host"
     ])
 
+    # Add the api_request_uuid header only if it is available
```

Comment:
I don't think the comment is necessary. The code is straight forward enough that it is clear what is happening without it.